Created plugin-pyenv-pipeline.yml

cstarner opened the pull request:
# Requested Verifications
## Repository
https://github.com/jenkinsci/pyenv-pipeline-plugin

## Proof of Commit Permissions
https://github.com/jenkinsci/pyenv-pipeline-plugin/graphs/contributors

## Link to Resolved HOSTING Issue
https://issues.jenkins-ci.org/browse/HOSTING-421

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

daniel-beck commented on permissions/plugin-pyenv-pipeline.yml:
```diff
@@ -0,0 +1,6 @@
+---
+name: "pyenv-pipeline"
+paths:
+- "org.jenkins-ci.plugins/pyenv-pipeline"
```
Group ID is incorrectly specified here.

Also, a group ID of `com.github`, unless you're working for the company, doesn't really make sense:
https://github.com/jenkinsci/pyenv-pipeline-plugin/blob/bead4b317f3f45e2acbd61ffd292d709a10bac1b/pom.xml#L11

I recommend:

* Choose a different group ID, e.g. the default of `org.jenkins-ci.plugins`
* Read this repository's README to learn how to map pom.xml values to the 'paths' entry in this YAML file.

cstarner commented:
I have changed the `groupId` to `org.jenkins-ci.plugins`, as recommended. I believe this also fixed the issue with the `paths` (since the value now matches the `groupId/artifactId` of the project)

daniel-beck commented:
@cstarner 

> I believe this also fixed the issue with the `paths`

Not really or I wouldn't have pointed you to the README. I'll fix it up.
